capstone-project/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
```

Comment:
I know this was autogenerated from the create project CLI, but can you remove the reference to logo